### PR TITLE
Remove Python 2 bits and pieces

### PR DESCRIPTION
**Pull request by m-horky**
* Card ID: ENT-4277: Drop `__future__` imports
* Card ID: ENT-4326: Drop `# -*- coding` comment
* Card ID: ENT-4302: Remove "u" prefix from strings

None of the changes should alter behaviour in any way. They have been used in past to ensure Python 2 - Python 3 compatibility.

### CODE REVIEW

File: src/subscription_manager/scripts/rhsmcertd_worker.py [hunk 1]
```diff
@@ -1,8 +1,6 @@
 # -*- coding: utf-8 -*-
```
Comment (jirihnidek) [inline src/subscription_manager/scripts/rhsmcertd_worker.py:1]: This line should be also removed.

**Review by jirihnidek — CHANGES_REQUESTED** [review]
Hi, thanks for PR.

I can still find some string that should be removed:

```
[jhnidek@thinkpad-t580 src]$ grep -R "coding: utf-8" *
rhsmlib/facts/pkg_arches.py:# -*- coding: utf-8 -*-
subscription_manager/repolib.py:# -*- coding: utf-8 -*-#
subscription_manager/scripts/rhsmcertd_worker.py:# -*- coding: utf-8 -*-
subscription_manager/repofile.py:# -*- coding: utf-8 -*-#
```

Note: Three PRs would be better for review ;-). Leave it as it is now, but try to avoid creating big PR in the future.

**Comment by m-horky** [comment]
Oops, I thought I haven't missed any. Thanks for notifying me.

You can open the Commits tab and do the review for each commit separately. Doing it independently could mean merge conflicts though... I'll do it in separate PRs next time something like this comes up.

**Review by jirihnidek — APPROVED** [review]
ACK

**Pull request merged**